python_package/interface.cpp

```diff
@@ -122,25 +122,26 @@ Status::type calc_twiss_wrapper (
     Status::type status = calc_twiss(
         accelerator, fixed_point, m66, twiss_, twiss0);
 
+    if (status != Status::success) return status;
     for (unsigned int i=0; i<n1; ++i){
         unsigned int j = i*n2;
         twiss[j] = twiss_[i].spos;
-        j++; twiss[j] = twiss_[i].betax;
```

Comment:
:open_mouth:

Reply:
now it woked using `twiss[++j]` !! I must've done something wrong before!

Reply:
@xresende, should I consider this an approval? :thinking:

Reply:
the bug was this first j++; right ?

Reply:
no, the bug was corrected in my firs commit, it was related to trying to fill the numpy array when `status`, returned by `calc_twiss`, was not `Status::success`. I just changed this sintax here because I think it is prettier this way!

Reply:
No, the way it is now is correct and the way it was before was also correct. Look at lines 127 and 128:
```
        unsigned int j = i*n2;
        twiss[j] = twiss_[i].spos;
```
I define `j` and then I use it once, before incrementing the first time.